src/fsharp/Fsc/fsc.exe.config

```diff
@@ -5,13 +5,12 @@
     <gcServer enabled="true"/>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
-        <assemblyIdentity
-          name="FSharp.Core"
-          publicKeyToken="b03f5f7f11d50a3a"
-          culture="neutral"/>
-        <bindingRedirect
-          oldVersion="2.0.0.0-4.4.1.0"
-          newVersion="4.4.1.0"/>
+        <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
+        <bindingRedirect oldVersion="2.0.0.0-4.4.1.0" newVersion="4.4.1.0"/>
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-1.2.1.0" newVersion="1.2.1.0" />
```

Comment:
Looking at the diff, paket would solve the one for `System.Collections.Immutable`, and if we made a nupkg of FSharp.Core for consumption in other projects, it could also write that one automatically.